helm/easeprobe/values.yaml

```diff
@@ -55,7 +55,7 @@ prometheus:
     enabled: false
   prometheus-pushgateway:
     enabled: false
-  
+
 ## Refer https://artifacthub.io/packages/helm/grafana/grafana
 grafana:
```

Comment:
Consider adding comments to the values.yaml file to explain what each configuration does. This can improve the usability of your Helm chart. [medium]